src/NuGetGallery.Core/GitHub/NuGetPackageGitHubInformation.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace NuGetGallery.GitHub
+{
+    public class NuGetPackageGitHubInformation
+    {
+        public NuGetPackageGitHubInformation() : this(0, null)
```

Comment:
Does anyone use this? If not, please remove

Reply:
Yes it's being used to actually return a valid NuGetPackageGitHubInformation object even if the package doesn't exist (has 0 TotalRepos)